cypress-tests/cypress/e2e/configs/Payment/Elavon.js

```diff
@@ -1,6 +1,15 @@
 import { customerAcceptance } from "./Commons";
 
 const successfulNo3DSCardDetails = {
+  card_number: "4111111111111111",
+  card_exp_month: "06",
+  card_exp_year: "50",
+  card_holder_name: "joseph Doe",
+  nick_name: "Bank1",
+  card_cvc: "123",
+};
```

Comment:
we can infact, do this in a far better way instead of duplicating it.

```suggestion
const cardDetailsWithoutNickName = {
  card_number: "4000000000002701",
  card_exp_month: "01",
  card_exp_year: "50",
  card_holder_name: "joseph Doe",
  card_cvc: "123",
};

// make sure to follow the order just so that `spread operator` works as expected in the subsequent calls made
const successfulNo3DSCardDetails = {
  ...cardDetailsWithoutNickName,
  nick_name: "Bank1",
};
```

verify it once, do the same everywhere. this greatly reduce code duplication!